tools/clang/test/CodeGenHLSL/literals_Mod.hlsl

```diff
@@ -49,6 +49,8 @@ float test() :Foo {
   VERIFY_TYPES(uint64_t, overload1(2ULL));
   VERIFY_TYPES(int64_t, overload1(2ll));
   VERIFY_TYPES(int64_t, overload1(2LL));
+  VERIFY_TYPES(float, overload1(1.0h));
+  VERIFY_TYPES(float, overload1(1.0H));
```

Comment:
Keep half here.  This would fail once half is different from float, requiring a half overload1 function to be added making it succeed again